varanny.go

```diff
@@ -247,7 +255,7 @@ func handleConnection(conn net.Conn, p *program) {
 
 		close(dbfsLevels)
 		close(cmdChannel)
-		//		close(stop)
```

Comment:
I also decided to uncomment this. I know the channel will be garbage collected, but the symmetry is nice, and, if the stop event is ever extended it'll be nice to just have this bit in place

Reply:
Will I have to do some regression testing on Windows. The killing of process gave me a lot of grief. That coupled with being a n00b golang programmer which means I sometimes have not idea what I'm doing :)